lib/autobuild/import/git.rb

```diff
@@ -272,7 +272,7 @@ def self.git_dir(package, require_working_copy)
         def self.validate_git_dir(package, require_working_copy, dir, style)
             if !style
                 raise ConfigException.new(package, 'import', retry: false),
-                    "while importing #{package.name}, #{package.importdir} does not point to a git repository"
+                    "while importing #{package.name}, #{package.importdir} does not point to a git repository. It seems the repository type changed, so remove the local directory and try again."
```

Comment:
"It may be that the repository type changed for this package. To solve this, remove the local directory and try again" would sound better to me. If you agree, I'll just do the change on the autoproj 2.0 branch manually, no need to update the PR.